### ISSUE TITLE
showcase: critical dependency warning on prettier/standalone.js

### ISSUE THREAD

**Issue by RobbyDeLaet** [NationalBankBelgium/stark#559]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/NationalBankBelgium/stark/blob/master/CONTRIBUTING.md#got-a-question-or-problem
</code></pre>

## Current behavior

When building the Showcase App, there is a Critical dependency warning on prettier/standalone.js

```
WARNING in ./node_modules/prettier/standalone.js 7277:15-66
Critical dependency: the request of a dependency is an expression
    at CommonJsRequireContextDependency.getWarnings (C:\LocalData\Dev\Workspace\stark-monorepo\showcase\node_modules\webpack\lib\dependencies\ContextDependency.js:40:18)
    at Compilation.reportDependencyErrorsAndWarnings (C:\LocalData\Dev\Workspace\stark-monorepo\showcase\node_modules\webpack\lib\Compilation.js:1315:24)
    at Compilation.finish (C:\LocalData\Dev\Workspace\stark-monorepo\showcase\node_modules\webpack\lib\Compilation.js:1126:9)
    at hooks.make.callAsync.err (C:\LocalData\Dev\Workspace\stark-monorepo\showcase\node_modules\webpack\lib\Compiler.js:545:17)
    at _done (eval at create (C:\LocalData\Dev\Workspace\stark-monorepo\showcase\node_modules\tapable\lib\HookCodeFactory.js:24:12), <anonymous>:9:1)
    at _err3 (eval at create (C:\LocalData\Dev\Workspace\stark-monorepo\showcase\node_modules\tapable\lib\HookCodeFactory.js:24:12), <anonymous>:56:22)
    at _addModuleChain (C:\LocalData\Dev\Workspace\stark-monorepo\showcase\node_modules\webpack\lib\Compilation.js:1054:12)
    at processModuleDependencies.err (C:\LocalData\Dev\Workspace\stark-monorepo\showcase\node_modules\webpack\lib\Compilation.js:980:9)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
 @ ./node_modules/@nationalbankbelgium/stark-ui/fesm5/stark-ui.js
 @ ./src/app/app.module.ngfactory.js
 @ ./src/main.browser.ts
```

## Expected behavior

The warning should be gone

## Minimal reproduction of the problem with instructions

Build the Showcase App

## What is the motivation / use case for changing the behavior?

The less warnings, the better

## Environment

<pre><code>
Angular version: 6.0.9

**Comment by christophercr** [comment]
There is actually an issue about this: https://github.com/prettier/prettier/issues/4959

**Comment by christophercr** [comment]
Now that the issue has been resolved in the latest release v1.16, we just need to upgrade 😉
